lib/hsa/mcwamp_hsa.cpp

```diff
@@ -137,6 +134,10 @@ int HCC_PROFILE_VERBOSE=0x1F;
 
 char * HCC_PROFILE_FILE=nullptr;
 
+int HCC_QUEUE_FLUSHING_RATIO=50;
+
+double QUEUE_FLUSHING_FRAC;
```

Comment:
nit: all calculation based on this value seems to be in `float`?

Reply:
I believe you are suggesting to have only one variable instead of the later extra step of `QUEUE_FLUSHING_FRAC = HCC_QUEUE_FLUSHING_RATIO / 100.0`?  I could see having the user specify a value between 0 and 1, e.g., 0.5, 0.4.

Otherwise I took your comment to say use `float` instead of `double`, although I thought the default without the `f` suffix was to use `double`, so the later calculations that use `QUEUE_FLUSHING_FRAC` would be using `double` anyhow.

Reply:
yes, it'd be better if we could remove this global variable, or at least keep it an integer.